generator/src/main/java/com/scottlogic/deg/generator/generation/string/AutomatonUtils.java

```diff
@@ -1,16 +1,41 @@
-package com.scottlogic.deg.generator.generation;
+package com.scottlogic.deg.generator.generation.string;
 
 import dk.brics.automaton.Automaton;
+import dk.brics.automaton.RegExp;
 import dk.brics.automaton.State;
 import dk.brics.automaton.Transition;
 
 import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
 class AutomatonUtils {
 
-    private static final char printableChar = ' ';
+    private static final Map<String, String> PREDEFINED_CHARACTER_CLASSES = instantiatePredefinedCharacterClasses();
+
+    private static final Pattern PATTERN_REQUOTED = Pattern.compile("\\\\Q(.*?)\\\\E");
```

Comment:
what is this?

can this and pattern_special be given more descriptive names

Reply:
These, along with anything else added to this file, are extracted from RegexStringGenerator with the same names as before.

I welcome any suggestions for better names of this field.

Reply:
i was hoping you knew what they did :p